CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# CONTRIBUTING GUIDELINES
+
+[日本語のコントリビューティングガイドライン](CONTRIBUTING_ja.md)
+
+## About of decomoji add or remove
+
+It does not accept in principle.
+
+Use decomoji-template.psd if you want to use new one.
+
+Remove files from `decomoji/` directory if you don't want to use one.
```

Comment:
Also, if you would like to remove a decomoji, you can do by removing a file from `decomoji/`